tool/jt.rb

```diff
@@ -1761,6 +1767,22 @@ def install_graal
     puts "$ #{TRUFFLERUBY_DIR}/tool/jt.rb ruby --graal ..."
   end
 
+  def build_native_image
+    build
+    java_home = install_jvmci
+
+    puts 'Building TruffleRuby native binary'
+    graal = Utilities.find_or_clone_repo('https://github.com/graalvm/graal.git')
+    chdir("#{graal}/substratevm") do
+      raw_sh 'git', 'fetch'
+      raw_sh 'git', 'checkout', Utilities.truffle_version
```

Comment:
This setup of 3 lines (clone,fetch,checkout) could be shared with `install_graal`. Could you extract it in a method?
The `chdir` can be just to `graal` for these operations.